### DISCUSSION TITLE
[Question] Is there a way to generate all tailwind classes at runtime ?

### DISCUSSION THREAD

**Discussion by flagrede** [tw-in-js/twind#132] [Q&A] [answered]
As of now only used css classes are generated at runtime (which is the expected behavior for production).
However, during development, it is really handy to have all css classes available in the devtools. Like using tailwind without purgecss.

![image](https://user-images.githubusercontent.com/530644/109391931-d33bff80-7919-11eb-8cb2-0df9672b8d26.png)

Is there a way to do that with the current version?

**Comment by sastan** [comment, marked as answer]
Sadly not yet. We have some ideas on how this could be implemented but haven't had the time to evaluate these.

**Reply by flagrede** [reply]
Thanks for the answer!
